lib/Util.js

```diff
@@ -0,0 +1,56 @@
+class ArrayUtil {
+
+    static Reducer( arr, fn ) {
+        return arr.reduce( ( memo, item, i ) => ( { ...memo, ...fn( item, i ) } ), { } )
+    }
+
+    static ShuffleArray( arr ) {
```

Comment:
should probably just remove this until we use it.  I feel like I only use it for generating fake data.